inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -69,6 +69,9 @@ credentials_dir: "{{ inventory_dir }}/credentials"
 # Can also be set to 'cloud', which lets the cloud provider setup appropriate routing
 kube_network_plugin: calico
 
+# CNI plugins support
+enable_cni: true
```

Comment:
Why still mess with this? Just remove any option to disable this and always deploy the standard CNIs, this makes for a more predictable environment.

Reply:
yes